inflate.c

```diff
@@ -1350,10 +1339,15 @@ int32_t Z_EXPORT PREFIX(inflateCopy)(PREFIX3(stream) *dest, PREFIX3(stream) *sou
         copy->distcode = copy->codes + (state->distcode - state->codes);
     }
     copy->next = copy->codes + (state->next - state->codes);
-    if (window != NULL) {
-        ZCOPY_WINDOW(window, state->window, (size_t)1U << state->wbits);
+
+    /* window */
+    copy->window = NULL;
```

Comment:
Possibly need wrap `copy->window` creation in `if (state->window != NULL)` check.

Reply:
You are right, that part of the check is missing now, the old check was dual-purpose, checking both the allocation but also indirectly whether state->window is actually allocated in the first place.